api/src/application/user/user-service.js

```diff
@@ -102,8 +127,10 @@ module.exports = (bookshelf, emailService, renderTemplate) => {
     },
 
     findByEmailForOidc: function(email) {
-      return bookshelf.model('user').forge({ email_id: email.toLowerCase() }).fetch()
-        .then(user => user ? user.serialize({strictOidc: true}) : null);
+      return bookshelf.model('user').where({ email_id: email.toLowerCase() }).fetch()
+        .then(user => {
+          return user ? user.serialize({strictOidc: true}) : null
+        });
```

Comment:
I've noticed you made a couple of these formatting changes turning calls like `() => thing` into `() => { return thing }`.

When you make formatting changes with no logic changes it bloats the code review and gives me extra stuff to look through.